vpr/src/place/place.cpp

```diff
@@ -1218,6 +1234,13 @@ static bool update_annealing_state(t_annealing_state* state,
                                    const t_placer_costs& costs,
                                    const t_placer_opts& placer_opts,
                                    const t_annealing_sched& annealing_sched) {
+    t_draw_state* draw_state = get_draw_state_vars();
+    if (draw_state->list_of_breakpoints.size() != 0) {
+        //update temperature in the current information variable
+        current_info_p.temp_count++;
```

Comment:
Need to move current_info into a breakpoint state class with a better name.  That class should be accessible from draw_state, or you should have a different get_bp_state() call to get a pointer to the top-level data class (either could work, and either avoids creating a new global variable that is accessible from anywhere which is bad).

Reply:
resolved